pkg/kubevirt/apis/provider_spec.go

```diff
@@ -16,21 +16,22 @@ package api
 
 import (
 	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
 	kubevirtv1 "kubevirt.io/client-go/api/v1"
 )
 
 // KubeVirtProviderSpec is the spec to be used while parsing the calls.
 type KubeVirtProviderSpec struct {
+	// Resources defines requests and limits resources of VMI
+	Resources *kubevirtv1.ResourceRequirements `json:"resources"`
 	// SourceURL is the HTTP URL of the source image imported by CDI.
 	SourceURL string `json:"sourceURL,omitempty"`
 	// StorageClassName is the name which CDI uses to in order to create claims.
+	// +optional
```

Comment:
remove all `+optional' as there is no code generator

Reply:
I would suggest to keep them, for documentation purposes, and ensure that they are placed correctly.